integrations/jina.md

```diff
@@ -3,11 +3,11 @@ layout: integration
 name: Jina AI
 description: Use the latest Jina AI embedding models
 authors:
-    - name: deepset
-      socials:
-        github: deepset-ai
-        twitter: deepset_ai
-        linkedin: deepset-ai
+  - name: deepset
```

Comment:
Looks like some spaces or tabs were removed here.. the frontmatter can be quite picky when generating the page.. Any chance we can revert the changes to the frontmatter since the content is the same?

Reply:
Thanks for the info, I have reverted.